build/bin/build-client-rpm

```diff
@@ -14,18 +14,26 @@ main() {
         set -o errexit
         set -o xtrace
 
+        # Install missing packages for Ubuntu 20.04
+        if [ -f /etc/os-release ] \
+        && grep -q 'Focal Fossa' /etc/os-release; then
+            export DEBIAN_FRONTEND=noninteractive
+            apt update
+            apt install -y sudo lsb-release
+        fi
+
         export pmm_version=$pmm_version
         export pmm_release=$pmm_release
         export sudo_path=\$(ls /usr/bin/sudo)
         mkdir -p /tmp/pmm
-        pushd /home/builder/results
+        cd /home/builder/results
```

Comment:
why remove pushd pop?

Reply:
The've updated `dash` to make it more strict for POSIX commands. When it called as `/bin/sh` it doesn't support `pushd`/`popd`. These internal commands are only available when binary is called as `/bin/bash`.

Reply:
okay thank you :+1: